contributors/devel/e2e-tests.md

```diff
@@ -79,6 +79,11 @@ getting started steps for your chosen cloud platform (which might involve
 changing the `KUBERNETES_PROVIDER` environment variable to something other than
 "gce").
 
+You can quickly recompile the e2e testing framework via `go install ./test/e2e`.
+This will not do anything besides allow you to verify that the go code compiles.
+If you want to run your e2e testing framework without re-provisioning the e2e setup,
+you can do so via `make WHAT=test/e2e/e2e.test`, and then re-running the ginkgo tests.
```

Comment:
i never ran e2e tests for nodes before, so it may be better to let someone experienced changing the doc.